src/UI/Art/SpriteArtist/SpriteUIUtils.java

```diff
@@ -108,6 +117,51 @@ public static BufferedImage makeTextMenu(Color bg, Color frame, Color focus, Arr
     return menuImage;
   }
 
+  /**
+   * Returns an image with the input string printed within the specified width, in normal text.
+   * @param reqWidth: Actual UI size in pixels that you want to fit the text into.
+   */
+  public static BufferedImage paintTextNormalized(String prose, int reqWidth)
+  {
+    // Figure out how big our text is.
+    int drawScale = SpriteOptions.getDrawScale();
+    int characterWidth = SpriteLibrary.getLettersUppercase().getFrame(0).getWidth() * drawScale;
+    int characterHeight = SpriteLibrary.getLettersUppercase().getFrame(0).getHeight() * drawScale;
+
+    ArrayList<String> lines = new ArrayList<String>();
+    // Unload our prose into the lines it already has
+    lines.addAll(Arrays.asList(prose.split("\\R"))); // \R matches all newline formats, yay convenience
+
+    for( int i = 0; i < lines.size(); ++i ) // basic for, since we care about indices
+    {
+      String line = lines.get(i);
+      if( line.length() * characterWidth <= reqWidth ) // if the line's short enough already, don't split it further
+        continue;
+
+      // TODO: check for word boundaries
```

Comment:
I'm not a huge fan of splitting words across lines. This is a good TODO to get done.